docs/usage/help.rst.inc

```diff
@@ -195,11 +195,30 @@ Examples::
         # susan is a nice person
         # include susans home
         + /home/susan
+        # also back up this exact file
+        + pf:/home/bobby/specialfile.txt
         # don't backup the other home directories
         - /home/*
         # don't even look in /proc
         ! /proc
 
+    As another example, you could do the same thing in either of these ways::
+
+        # specified on command line
+        borg create --exclude /home/bobby/junk repo::arch /home/bobby /home/susan
+        # specified in patternfile
+        borg create --patterns-from patternfile.lst repo::arch
+
+    The patternfile::
+
+        R /home/bobby
+        R /home/susan
+
+        # note that excludes use fm: by default and patternfiles use sh: by default.
+        - /home/bobby/junk
+
```

Comment:
you say "same thing", so people might expect this is precisely doing the same thing as the previous example, which it does not.

maybe adjust it so it really does exactly the same thing?